src/algebra/ring/add_aut.lean

```diff
@@ -0,0 +1,30 @@
+/-
+Copyright (c) 2022 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import group_theory.group_action.group
+import algebra.module.basic
+
+/-!
+# Multiplication on the left/right as additive automorphisms
+
+In this file we define `add_aut.mul_left` and `add_aut.mul_right`.
```

Comment:
Maybe
```suggestion
In this file we define `add_aut.mul_left` and `add_aut.mul_right`.

See also `add_monoid.End.mul_left` and `add_monoid.End.mul_right` for multiplication by `R` instead of `Rˣ`.
```